apps/video/style/video.css

```diff
@@ -443,21 +442,21 @@ header .icon:after {
 /* video player controls */
 #videoControls {
   position: absolute;
+  display: block;
   top: 0;
   left: 0;
   width: 100%;
   height: 100%;
   z-index: 10;
   opacity: 1;
-  will-change: opacity;
-  -moz-transition: opacity 0.5s;
+  visibility: visible;
```

Comment:
TBD as to whether this change is necessary based on whether 'hide-controls' element is necessary.